spec/lib/bandiera/client_spec.rb

```diff
@@ -4,7 +4,41 @@
   let(:base_uri)  { 'http://bandiera.com' }
   let(:api_uri)   { "#{base_uri}/api" }
   let(:logger)    { double.as_null_object }
-  subject         { Bandiera::Client.new(api_uri, logger) }
+  let(:client_instance) { Bandiera::Client.new(api_uri, logger) }
```

Comment:
Not really sure on this change - `subject` is the standard name for the test target...